boards/pine64-pinecube/default.nix

```diff
@@ -0,0 +1,17 @@
+{
+  device = {
+    manufacturer = "PINE64";
+    name = "PineCube";
+    identifier = "pine64-pinecube";
+    productPageURL = "https://www.pine64.org/cube/";
+  };
+
+  hardware = {
+    soc = "allwinner-s3";
+  };
```

Comment:
I believe simply adding the SPI setting here would be enough to make it a valid build. Allwinner targets are extremely simple to support.

Reply:
I'm reluctant to add this without testing, and testing it is not quite at the top of my priority list. Eventually though :)

Reply:
Good mindset. I'll dig in my box of projects and get mine out to test it.

Note that on Allwinner it's also basically impossible to "brick", since it will always prefer starting from SD. So testing the installer should be a trivial matter.